client/package.json

```diff
@@ -14,6 +14,7 @@
     "react": "^16.13.1",
     "react-copy-to-clipboard": "^5.0.2",
     "react-dom": "^16.13.1",
+    "react-gtm-module": "^2.0.8",
```

Comment:
hey @MohamedEl1 just wondering, did you install through docker? (see readme) 
since the package-lock wasn't added, it should also be committed to git when updating dependencies

Reply:
Good catch,  will update it